EIPS/eip-6160.md

```diff
@@ -0,0 +1,80 @@
+---
+eip: 6160
+title: Multi-chain Native Tokens
+description: An interface for multi-chain native tokens.
+author:  Seun Lanlege (@seunlanlege), Sam Omidiora (@samparsky)
+discussions-to: https://ethereum-magicians.org/t/eip-6160-multichain-token-standard/12769
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-01-31
+requires: 20, 5679, 5982
+---
+
+## Abstract
+
+We propose new [`ERC20`](./eip-20.md), [`ERC721`](./eip-721.md) and [`ERC1155`](./eip-1155.md) extension interfaces which will allow provably secure, trustless bridge contracts to mint & burn tokens, enabling a new class of `ERC` tokens that can be native to multiple chains.
+
+## Motivation
+
+[EIP-4844](./eip-4844.md) will set the stage for the Cambrian explosion of L2s on Ethereum. With multiple L2s living on the ethereum blockchain, It therefore becomes necessary to standardize an interface that will allow for native `ERC` tokens to flow across these L2s, removing the need for wrapped representations of these assets.
+
+Optimisim and Arbitrum, optimistic L2s which are already live on Ethereum today currently have their own custom token interfaces¹ that allow their bridge contracts to mint & burn tokens on L1 & L2. In order to remove the cognitive load on token issuers & developers from extending multiple interface for all the various L2s that will live on Ethereum. It becomes necessary to define a standardized API that all token issuers should implement in order to provide these trustless bridge contracts with the interface needed to transport tokens across these different chains.
```

Comment:
We only allow very specific external references (see [EIP-1](https://eips.ethereum.org/EIPS/eip-1#linking-to-external-resources).)

```suggestion
Optimisim and Arbitrum, optimistic L2s which are already live on Ethereum today currently have their own custom token interfaces that allow their bridge contracts to mint & burn tokens on L1 & L2. In order to remove the cognitive load on token issuers & developers from extending multiple interface for all the various L2s that will live on Ethereum. It becomes necessary to define a standardized API that all token issuers should implement in order to provide these trustless bridge contracts with the interface needed to transport tokens across these different chains.
```